### DISCUSSION TITLE
Question on Plotter() camera viewing angles

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#706] [Q&A] [answered]
Hi Marco! 
Just a quick question: I want to take screenshot of my plot from different angles automatically. When the plot initiates, it is looking directly down on the plot. How would I make it switch to an axiomatic view using the show() call?
Thanks, 
j

**Comment by marcomusy** [comment, marked as answer]
Hi, what I would do, after the first call to `show()` use a loop:
```python
...
plt.show(...)
for i in range():
    plt.camera.Azimuth(1) # and/or Elevation()
    plt.render()
```
Or use the `plt.move_camera()` method.
One other option is `examples/other/makeVideo.py`
